src/WebApiToTypeScript/Interfaces/InterfaceService.cs

```diff
@@ -334,7 +334,7 @@ private List<MemberWithCSharpType> GetMembers(TypeDefinition typeDefinition)
                 });
 
             var properties = typeDefinition.Properties
-                .Where(p => p.GetMethod != null && p.GetMethod.IsPublic && !p.IsSpecialName)
+                .Where(p => p.GetMethod != null && p.GetMethod.IsPublic && !p.IsSpecialName && !Helpers.HasCustomAttribute(p, "JsonIgnoreAttribute"))
```

Comment:
Might be better to have a config, something like:
"IgnorePropertiesWithAttributes" and then specify attributes - those properties with those attributes will be ignored?
Might be better than having a magic string